.circleci/config.yml

```diff
@@ -248,11 +248,11 @@ jobs:
           command: emulator -accel-check
 
       - run:
-          name: Run e2e tests
+          name: Run build verification e2e tests
           command: |
             set -euo pipefail
             cd packages/mobile
-            yarn run test:e2e:android
+            yarn run test:e2e:android -t "#BVT"
```

Comment:
Ideally, I'd prefer we run all tests all the time. We don't want any regression to land on main.
Even if it takes a bit more time.

Reply:
I agree we should make tests faster and run them all the time but this tag feature might be useful if we parallelize running the tests right?

Reply:
This feature won't impact test paralyzation. It's used to tag certain tests as build verification tests (BVT) to run on every commit to an active PR. I've gone ahead and removed that flag from the tests and the Circle Ci config so that all tests will run on commits to active PR's. In the future as the test suite grows in size this might be a feature we want to revisit.

This PR still is useful for running for local development enabling the running of individual files or test specs via the `-f` and `-t` options respectively.